quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMapper.java

```diff
@@ -186,6 +193,7 @@ public String getTo() {
         private boolean isBuildTimeProperty = false;
         private boolean isMasked = false;
         private ConfigCategory category = ConfigCategory.GENERAL;
+        private String paramLabel;
```

Comment:
perhaps default to "none" / NO_PARAM_LABEL? then you could simplify https://github.com/keycloak/keycloak/pull/8713/files#diff-14281ffb75b6fe566e89207ce2787a0f727cacbcae7ab0d0c7e54a3de5910233R70 i think. but not sure, bc we also have manual properties, so the isBlank would still be needed.

Reply:
Please, note that this is not used at all in our mappers. This is basically trying to avoid Picocli setting a default value (based on field/method name) for options explicitly defined in commands.